quarkus/maven-plugin/pom.xml

```diff
@@ -0,0 +1,62 @@
+
+<!--
+  ~ Copyright 2016 Red Hat, Inc. and/or its affiliates
+  ~ and other contributors as indicated by the @author tags.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+        <artifactId>keycloak-docs-parent</artifactId>
+        <groupId>org.keycloak</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../../operator/pom.xml</relativePath>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Keycloak Operator Maven Plugin</name>
+    <artifactId>keycloak-operator-maven-plugin</artifactId>
```

Comment:
I'm wondering if makes more sense to:

* Create a parent pom for the operator
* Both the plugin and operator inheriting from this parent pom

The reason is that this module is really specific to the operator and having it at the root is a bit confusing because it has no relation to the distribution itself.

In fact, this module is inheriting from the operator pom already.

Reply:
I completely agree 👍  the `maven-plugin` is here only for "demo" purposes